src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reverseRobot = {};
+
+  for (const key in robot) {
+    if (reverseRobot.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      reverseRobot[robot[key]] = key;
+    }
```

Comment:
you don't need `else` block here,
```suggestion
    }
   
    reverseRobot[robot[key]] = key;
```